### ISSUE TITLE
Replace pianoroll with `FrameSequence`, rows with `Frame`

### ISSUE THREAD

**Issue by vabold** [TASLabz/mkw-scripts#11]
As a rough sketch of what our new infrastructure would look like:
```py
class Frame:
    def __init__(self, raw: List):
        # Initialize frame information based on the CSV line


class FrameSequence:
    def __init__(self, filename: str):
        self.frames = []          # List of initialized Frame classes
        self.filename = filename  # Name of the CSV file

        self.refresh()

    def refresh(self) -> None:
        # Load the CSV and iterate self.process() over each line

    def process(self, raw_frame: List) -> None:
        # Process raw frame (list from reading a line in the CSV) and add it to the sequence
```

**Comment by ximk** [comment]
Resolved in #10.

**Comment by vabold** [comment]
Resolved in 2994858c3ddf5e4346b0a4b6963daaf27c069ef5.
